test/e2e-v2/cases/airflow/airflow-main/docker-compose.yaml

```diff
@@ -0,0 +1,286 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  # SkyWalking OAP server
+  oap:
+    extends:
+      file: ../../../script/docker-compose/base-compose.yml
+      service: oap
+    ports:
+      - 12800:12800
+    entrypoint: ['sh', '-c', '/download-mysql.sh /skywalking/oap-libs && chmod 777 /skywalking/docker-entrypoint.sh && /skywalking/docker-entrypoint.sh']
+    networks:
+      - e2e
+    environment:
+      - TZ=Asia/Shanghai
+      - SW_STORAGE=mysql
+      - SW_DATA_SOURCE_USER=root
+      - SW_DATA_SOURCE_PASSWORD=password
+      - SW_JDBC_URL=jdbc:mysql://mysql:3306/swtest?allowMultiQueries=true
+      - SW_CORE_TOPN_REPORT_PERIOD=2
+    depends_on:
+      mysql:
+        condition: service_healthy
+
+  # SkyWalking UI
+  ui:
+    image: xiaodong12315/skywalking-ui:2.0
+    ports:
+      - "8082:8080"
+    networks:
+      - e2e
+    environment:
+      - SW_OAP_ADDRESS=http://oap:12800
+      - TZ=Asia/Shanghai
+    depends_on:
+      oap:
+        condition: service_healthy
+
+  # MySQL database
+  mysql:
+    image: mysql:8.0.13
+    networks:
+      - e2e
+    volumes:
+      - ./my.cnf:/etc/my.cnf
+    environment:
+      - "MYSQL_ROOT_PASSWORD=password"
+      - "MYSQL_DATABASE=swtest"
+    ports:
+      - 3306
+    healthcheck:
+      test: ["CMD", "bash", "-c", "cat < /dev/null > /dev/tcp/127.0.0.1/3306"]
+      interval: 5s
+      timeout: 60s
+      retries: 120
+
+  # OpenTelemetry collector 1
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.96.0
+    networks:
+      - e2e
+    command: [ "--config=/etc/otel-collector-config.yaml" ]
+    volumes:
+      - ../airflow-exporter/otel-collector-config.yaml:/etc/otel-collector-config.yaml
+    expose:
+      - 55678
+    depends_on:
+      oap:
+        condition: service_healthy
+    environment:
+      - TZ=Asia/Shanghai
+
+  # OpenTelemetry collector 2
+  otel-collector2:
+    image: otel/opentelemetry-collector-contrib:0.96.0
+    networks:
+      - e2e
+    command: [ "--config=/etc/otel-collector-config.yaml" ]
+    volumes:
+      - ../airflow-exporter/otel-collector-config2.yaml:/etc/otel-collector-config.yaml
```

Comment:
Do we have to use two `otel-collector` with different configs? Users usually deploy only one to collect all services's metrics.